Vagrantfile

```diff
@@ -50,6 +50,22 @@ Vagrant.configure("2") do |config|
     end
   end
 
+  config.vm.define "jenkins-node-debian10" do |override|
+    override.vm.hostname = "jenkins-node-debian10"
+    override.vm.box = "debian/buster64"
+
+    override.vm.provider "libvirt" do |libvirt|
+      libvirt.memory = "4096"
+    end
+    config.vm.provision "install puppet", type: "shell", inline: <<-SHELL
+      . /etc/os-release
+      wget https://apt.puppet.com/puppet6-release-${VERSION_CODENAME}.deb
+      apt-get install -y ./puppet6-release-${VERSION_CODENAME}.deb
```

Comment:
TIL: apt-get can install local files

Reply:
Yeah, and contrary to `dpkg` it will resolve dependencies when doing so ;)